projects/fsastorefrontlib/src/cms-components/assets-table/assets-table.component.html

```diff
@@ -54,5 +54,18 @@
   </table>
 </ng-container>
 <ng-template #noAssets>
-  <h3 class="text-center mt-5">{{ 'dashboard.noAssets' | cxTranslate }}</h3>
+  <div [ngSwitch]="assetSelected">
```

Comment:
Maybe there is no need for this Switch here... Could you perhaps try something like:
```
  <h3 class="text-center mt-5">
    {{
      (assetSelected === 'quotes'
        ? 'dashboard.noQuotesOrApplication'
        : assetSelected === 'policies'
        ? 'dashboard.noPolicies'
        : assetSelected === 'claims'
        ? 'dashboard.noClaims'
        : ''
      ) | cxTranslate
    }}
  </h3>
```